app/resources/v1/users.js

```diff
@@ -316,6 +318,8 @@ exports.get = async function (req, res) {
     }
   }
 
+  // TODO this function seems like it could be replaced by
```

Comment:
these notes I could take out but are breadcrumbs for potential future refactoring